### ISSUE TITLE
osgearth_conv "--extents" log

### ISSUE THREAD

**Issue by devil-darkside** [gwaldron/osgearth#2656]
osgEarth Version (required): 3.7

Description of the problem:
When osgearth_conv is exporting to TMS (for example) with specified "extents", the values are not reported in the log.
World extents are always shown.

**Comment by gwaldron** [comment]
This is true, though the tiles in the new TMS repository will only be ones that intersect the extents.
Is this causing a problem for you?

**Comment by devil-darkside** [comment]
Not really.

**Comment by gwaldron** [comment]
OK good because after a quick review this would be kind of a pain to fix TBH :)

**Comment by devil-darkside** [comment]
Ok, no problem...
After several runs I have understood the export is working well but I've opened the issue for future bug fixes.

**Comment by devil-darkside** [comment]
I've found another problem.
Since the XML file is always generated as a standard, if you load an elevation layer with all the levels defined this will cause a loading issue and the features to not be displayed at all.
E.g. feature to be clamped to the terrain and you have defined an elevation layer with all 19 levels in the XML file, the feature will not be shown since the XML file has empty levels.

**Comment by gwaldron** [comment]
Please open a new ticket with an example and more details -- I don't understand the issue as described.
